### PR TITLE
Shwetha/Rey - Bookview

### LINKED ISSUE #36: Book View Page
As a book user , I want to navigate to a unique page for each book that shows information about a particular book, such as author, publisher, genres, and title

**Tasks:**
- [x] Setup UI template for book page

**Acceptance Criteria:**
- [x] Users can navigate to a page that displays information about each individual book
- [x] Users can view all the information associated with that particular book

### PR DESCRIPTION
**Pull request by nacxitl**
Closes #36

### CODE REVIEW

**Review by tfurey96 — APPROVED** [review]
looks good

File: frontend/src/services/index.ts [hunk 1]
```diff
@@ -12,13 +12,15 @@ const BASE_URL = import.meta.env.DEV
 
 // TODO: Duplicate code
 export type IBook = {
+  average_rating: number | undefined;
```
Comment (CheesyGamer77) [inline frontend/src/services/index.ts:15]: Discussed in class
```suggestion
  average_rating: number;
```

**Review by CheesyGamer77 — APPROVED** [review]
One minor critique per discussion in class (all books have an average rating defined)

File: frontend/src/services/index.ts (hunk 1 above)
Comment (nacxitl) [reply frontend/src/services/index.ts]: Resolved in commit 259eaf0

**Pull request merged**